raiden/storage/sqlite.py

```diff
@@ -74,6 +116,14 @@ def assert_sqlite_version() -> bool:  # pragma: no unittest
     return True
 
 
+def adapt_ulid_identifier(ulid: ULID) -> bytes:
```

Comment:
why the specialized naming?
```suggestion
def ulid_to_bytes_identifier(ulid: ULID) -> bytes:
```
or even just
```suggestion
def ulid_to_bytes(ulid: ULID) -> bytes:
```

I would argue that the `adapt_` / `convert_` pattern is mentally hard...

Reply:
I see that these are nowhere to be used, so ignore...